octobot_trading/personal_data/orders/order_util.py

```diff
@@ -133,3 +133,9 @@ def parse_order_status(raw_order):
 
 def parse_is_cancelled(raw_order):
     return parse_order_status(raw_order) in {enums.OrderStatus.CANCELED, enums.OrderStatus.CLOSED}
+
+
+def log_order_creation_error(logger, error, order_type, symbol, quantity, price, stop_price):
```

Comment:
I don't think it's the proper place to put this method. The order creation error is linked to an exchange action, not an OctoBot internal error. If you don't want to keep it in `ccxt_exchange` that can be understood I think it should be put into an exchange util module or a parent of `ccxt_exchange`.